ercc/attestation/Makefile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TOP = ../..
+include $(TOP)/build.mk
+
+.PHONY: all clean .sgx_mode
```

Comment:
.PHONY: all and clean are already in build.mk thus not needed here.

Reply:
Yep, it's redundant, I have removed it.